engine/pkg/meta/common/client.go

```diff
@@ -11,27 +11,45 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package extension
+package common
```

Comment:
I think `common` is not a meaningful package name. Can we give a more descriptive name?

Reply:
It's for the placement of common inteface and structure for meta, although this name is too `common`.
I will try.

Reply:
Have Changed to 'model'.